CHANGELOG.md

```diff
@@ -1,3 +1,64 @@
+# v0.7.0
+
+## Enhancements
+
+* Remove timestamping authority (#813)
```

Comment:
Can we call this one out loudly, since it's a breaking change to the API?

Reply:
done PTAL, thanks for the review @haydentherapper